builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -1,6 +1,7 @@
 package vsphere
 
 import (
+	"errors"
```

Comment:
You probably know better than I, but should we just use `return fmt.Errorf("Error when converting string to bool: %s", err)` instead of pulling in `errors`.  I am seeing that pattern in most of the code.